### DISCUSSION TITLE
Feature Request: Export Dependency Graph

### DISCUSSION THREAD

**Discussion by jeremy-bridges** [community/community#18694] [Other Feature Feedback, Questions, & Ideas]
When reporting our security vulnerabilities at different times of year to our internal auditors, we need an artifact to document. Dependency-Check (https://owasp.org/www-project-dependency-check) is another tool we're considering in order to capture a point-in-time profile of our dependency graph. But, as this takes extra time to set up and maintain, we'd love to just use GitHub's tool that is already set up.

**Comment by courtneycl** [comment]
Hi @jeremy-bridges, what does exporting the dependency graph as an artifact look like to you? Download as a csv, or something else?
